drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -149,6 +149,102 @@ local child_device_profile_overrides = {
 
 local detect_matter_thing
 
+local CUMULATIVE_REPORTS_NOT_SUPPORTED = "__cumulative_reports_not_supported"
+local FIRST_EXPORT_REPORT_TIMESTAMP = "__first_export_report_timestamp"
+local EXPORT_POLL_TIMER_IS_SET = "__export_poll_timer_is_set"
+local EXPORT_REPORT_TIMEOUT = "__export_report_timeout"
+local TOTAL_EXPORTED_ENERGY = "__total_exported_energy"
+local LAST_EXPORTED_REPORT_TIMESTAMP = "__last_exported_report_timestamp"
+local RECURRING_EXPORT_REPORT_POLL_TIMER = "__recurring_export_report_poll_timer"
+local MINIMUM_ST_ENERGY_REPORT_INTERVAL = (15 * 60) -- 15 minutes, reported in seconds
+local SUBSCIPTION_REPORT_OCCURRED = "__s"
```

Comment:
Just wondering, is `__s` intentional or did you mean to use something like `__subscription_report_occurred`?

Reply:
no, good catch :+1: